--- ../main.c	2022-02-14 17:11:43.000000000 -0500
+++ ckstyle/main.c.gnu	2022-02-14 17:13:44.604997939 -0500
@@ -1,4 +1,4 @@
-#define  _GNU_SOURCE
+#define _GNU_SOURCE
 #include <stdio.h>
 
 #include <getopt.h>
@@ -6,12 +6,12 @@
 #include <stdint.h>
 #include <stdlib.h>
 
-#include "statemodel.h"
-#include "stringmodel.h"
 #include "intmodel.h"
-#include "valmodel.h"
 #include "objmodel.h"
 #include "parser.h"
+#include "statemodel.h"
+#include "stringmodel.h"
+#include "valmodel.h"
 
 // Used to specify which of the four FSMs to use
 typedef enum
@@ -42,45 +42,46 @@
       return EXIT_FAILURE;
     }
 
-  FILE * fp;
-  char * line = NULL;
+  FILE *fp;
+  char *line = NULL;
   size_t len = 0;
   size_t read;
   char *str = NULL;
   bool success = false;
   fp = fopen (filename, "r");
-    if (fp == NULL)
-        exit(EXIT_FAILURE);
-  while ((read = getline(&line, &len, fp)) != -1)
-  {
-    fsm_t *string = string_init (line);
-    switch (type)
-      {
-        case STR:
+  if (fp == NULL)
+    exit (EXIT_FAILURE);
+  while ((read = getline (&line, &len, fp)) != -1)
+    {
+      fsm_t *string = string_init (line);
+      switch (type)
         {
-          success = accept_string (string, &str);
-          if (success) 
-          {
-            printf ("STRING: '%s'\n", str);
-            printf ("Success!\n");
-            free (string);
-            free (line);
-            free (str);
-            return EXIT_SUCCESS;
-          } else
+        case STR:
           {
-            printf("Parsing %s failed\n", filename);
-            free (string);
-            free (line);
-            free (str);
-            return EXIT_FAILURE;
+            success = accept_string (string, &str);
+            if (success)
+              {
+                printf ("STRING: '%s'\n", str);
+                printf ("Success!\n");
+                free (string);
+                free (line);
+                free (str);
+                return EXIT_SUCCESS;
+              }
+            else
+              {
+                printf ("Parsing %s failed\n", filename);
+                free (string);
+                free (line);
+                free (str);
+                return EXIT_FAILURE;
+              }
+            break;
           }
-          break;
-        }
         default:
           break;
-      }
-  }
+        }
+    }
   fclose (fp);
   free (line);
   free (str);
