--- ../parser.c	2022-02-14 21:57:03.196496930 -0500
+++ ckstyle/parser.c.gnu	2022-02-15 14:30:59.323111266 -0500
@@ -25,49 +25,51 @@
 accept_string (fsm_t *fsm, char **result)
 {
   while (*fsm->current != '\0')
-  {
-    switch (*fsm->current)
     {
-      case '\"':
-      {
-        if (fsm->state == STR_INIT)
+      switch (*fsm->current)
         {
-          handle_event(fsm, OPEN_QUOTE);
-        } else
-        {
-          handle_event(fsm, CLOSE_QUOTE);
-        }
-        break;
-      }
-      case '\\':
-      {
-        handle_event(fsm, BACKSLASH);
-        fsm->current++;
-        if (*fsm->current == '\\' || *fsm->current == '\"')
-        {
-          handle_event(fsm, ESC_CHAR);
-        } else
-        {
-          handle_event(fsm, NO_ESC);
-          return false;
+        case '\"':
+          {
+            if (fsm->state == STR_INIT)
+              {
+                handle_event (fsm, OPEN_QUOTE);
+              }
+            else
+              {
+                handle_event (fsm, CLOSE_QUOTE);
+              }
+            break;
+          }
+        case '\\':
+          {
+            handle_event (fsm, BACKSLASH);
+            fsm->current++;
+            if (*fsm->current == '\\' || *fsm->current == '\"')
+              {
+                handle_event (fsm, ESC_CHAR);
+              }
+            else
+              {
+                handle_event (fsm, NO_ESC);
+                return false;
+              }
+            break;
+          }
+        default:
+          {
+            handle_event (fsm, NON_CTRL);
+            break;
+          }
         }
-        break;
-      }
-      default:
-      {
-        handle_event(fsm, NON_CTRL);
-        break;
-      }
+      fsm->current++;
     }
-    fsm->current++;
-  }
   if (fsm->length != 0)
-  {
-    char *temp = strdup(fsm->buffer);
-    *result = temp;
-    // result = &fsm->buffer;
-    return true;
-  }
+    {
+      char *temp = strdup (fsm->buffer);
+      *result = temp;
+      // result = &fsm->buffer;
+      return true;
+    }
   return false;
 }
 
