--- ../statemodel.h	2022-02-13 13:36:42.098920005 -0500
+++ ckstyle/statemodel.h.gnu	2022-02-14 17:13:41.088965060 -0500
@@ -15,7 +15,7 @@
 typedef struct fsm fsm_t;
 
 // All entry, exit, and effect instances use the action type
-typedef void (*action_t)(fsm_t *);
+typedef void (*action_t) (fsm_t *);
 
 // Generic FSM structure with pointers to the transition, effect, and
 // entry tables. Note that transitions and effects should point to
@@ -28,7 +28,7 @@
   size_t nevents; /* number of events for this FSM */
 
   /* pointer to the FSM's transition function */
-  state_t (*transition)(struct fsm *, event_t, action_t *, action_t *);
+  state_t (*transition) (struct fsm *, event_t, action_t *, action_t *);
 
   // Pointers to the input text, including the current byte being processed
   char const *input;
@@ -49,6 +49,6 @@
 };
 
 // Generic entry point for handling events
-void handle_event(fsm_t *fsm, event_t event);
+void handle_event (fsm_t *fsm, event_t event);
 
 #endif
